ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -391,14 +390,6 @@ struct FrameworkBuilder {
       }
       umbrellaHeader = umbrellaHeaderURL.lastPathComponent
     }
-    // Copy the Headers over.
-    let headersDestination = frameworkDir.appendingPathComponent("Headers")
-    do {
-      try fileManager.copyItem(at: headersDir, to: headersDestination)
-    } catch {
-      fatalError("Could not copy headers from \(headersDir) to Headers directory in " +
-        "\(headersDestination): \(error)")
-    }
```

Comment:
This was problematic as we were picking one platform framework, and copying the headers to a shared folder that we then used to re-construct each platform framework. As of Xcode 14, the generated -Swift.h header is architecture specific which has implications for each platform. Namely, using a -Swift.h header across all platforms is not a good practice.